src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunSinkV2.cs

```diff
@@ -0,0 +1,225 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using Serilog.Core;
+using Serilog.Events;
+using Mindscape.Raygun4Net;
+
+namespace Serilog.Sinks.Raygun;
+
+#if NET
+/// <summary>
+/// RaygunSink for Serilog which handles logging to Raygun.
+/// </summary>
+public class RaygunSinkV2 : ILogEventSink
```

Comment:
Minor: I'm usually not a fan of classes with version numbers. I imagine we'll eventually phase out the first version and then probably want to drop the version number from this one. I'm wondering if it could have a name that we don't later need to change. e.g. "RaygunClientSink". But I realize this is just a nit pick.

Reply:
Renamed to `RaygunClientSink`